test/export/InstanceExporter.test.ts

```diff
@@ -1755,6 +1755,62 @@ describe('InstanceExporter', () => {
       expect(exportedBundledPatient).toHaveLength(1);
     });
 
+    it('should only add optional children of list elements and the implied elements of those children to entries in the list that fix values on those children', () => {
+      // * generalPractitioner only Reference(Practitioner | Organization)
+      const onlyRule = new OnlyRule('generalPractitioner');
+      onlyRule.types.push(
+        {
+          type: 'Practitioner',
+          isReference: true
+        },
+        {
+          type: 'Organization',
+          isReference: true
+        }
+      );
+      // * generalPractitioner.extension contains
+      //   http://hl7.org/fhir/us/core/StructureDefinition/us-core-direct named direct-email 0..1
+      const containsRule = new ContainsRule('generalPractitioner.extension');
+      containsRule.items.push({
+        name: 'direct-email',
+        type: 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-direct'
```

Comment:
Instead of using this extension, we could just use on of the extensions that is already in the `testdefs/package` folder. That way we don't have to add another file, and we don't have to add a file from US Core. For example, the http://hl7.org/fhir/StructureDefinition/patient-mothersMaidenName extension is already in `testdefs/package`, and it should work pretty much analogously to the US Core direct email extension (just has a `valueString` instead of `valueBoolean`).